source/scripts/app.js

```diff
@@ -441,10 +450,14 @@ async function init() {
   createNavbar();
   createHomePage();
   createExplorePage();
+  populateExplorePage();
+  bindExploreLoadButton();
```

Comment:
The `populateExplorePage()` and `bindExploreLoadButton()` functions are already called lower down in the `init()` function
```suggestion
```